File: articles/framework-design-in-policy-research/fortran/policy_framework_metrics.f90
program policy_framework_metrics
  implicit none

  real :: objects
  real :: relationships

  objects = 10.0
  relationships = 10.0

  print *, "Policy Framework Metrics"
  print *, "Metadata coverage: ", 9.0 / objects
  print *, "Equity context coverage: ", 6.0 / objects
  print *, "Causal context coverage: ", 6.0 / objects
  print *, "Relationship traceability: ", 9.0 / relationships
  print *, "Underspecified relationship risk: ", 1.0 / relationships
end program policy_framework_metrics

File: articles/framework-design-in-policy-research/cpp/policy_framework_metrics.cpp
#include <iostream>
#include <iomanip>

int main() {
    double objects = 10.0;
    double relationships = 10.0;

    std::cout << std::fixed << std::setprecision(3);
    std::cout << "Metadata coverage: " << 9.0 / objects << "\n";
    std::cout << "Equity context coverage: " << 6.0 / objects << "\n";
    std::cout << "Causal context coverage: " << 6.0 / objects << "\n";
    std::cout << "Relationship traceability: " << 9.0 / relationships << "\n";
    std::cout << "Underspecified relationship risk: " << 1.0 / relationships << "\n";

    return 0;
}
